SpencerConstance/woocommerce
# Changed what triggered "capture_payment" on order status change.

Instead of having "capture_payments" trigger only when the status changed from "On-Hold" to "Completed", I changed it to always trigger when status is changed to completed.

### All Submissions:




### Changes proposed in this Pull Request:



Closes #23626. Fixes an issue where custom order status' stopped the PayPal gateway from capturing previously authorized orders when the status is changed to completed.

### How to test the changes in this Pull Request:

1. Add a custom order status.
2. Set the PayPal gateway to authorize only.
3. Create an order and change the order status to the custom order status.
4. Change the order status to completed, and check in PayPal if the transaction has been captured.

### Other information:




### Changelog entry
-Fixed an issue where custom order status' stopped the PayPal gateway from capturing orders when the status is changed to completed.
